pkg/cfn/builder/iam.go

```diff
@@ -0,0 +1,122 @@
+package builder
```

Comment:
The idea is to keep same types of resources grouped together, as we might compose IAM-only stack in the future

Reply:
I like this